### DISCUSSION TITLE
pac solution pack vs. solution packager

### DISCUSSION THREAD

**Discussion by ciaomatteo** [microsoft/powerplatform-build-tools#911] [Q&A] [answered]
Documentation says pac cli is the prefered way for creating solution zip files? Is solution packager going to be deprecated soon?

**Comment by petrochuk** [comment, marked as answer]
`pac cli` internally uses the same `SolutionPackagerLib` as stand-alone [Solution Packager](https://learn.microsoft.com/en-us/power-platform/alm/solution-packager-tool). There are no plans to deprecate this functionality. 

![image](https://github.com/microsoft/powerplatform-build-tools/assets/30735471/1fdcce7e-9b95-470e-84f0-a2cfcf60e531)
